### ISSUE TITLE
[Chromaticraft] Support for Dynmap?

### ISSUE THREAD

**Issue by Rubyheart** [ReikaKalseki/Reika_Mods_Issues#3125]
A very low priority feature request to be sure.  Is there any chance of adding support to Chromaticraft (and the other suite of mods) to Dynmap?  Currently, none of the leaf blocks render correctly in Dynmap, for example [this](https://i.redd.it/ntzbvwizve8a1.jpg) is a picture of a rainbow forest biome.  In game, all of the logs have leaves, but on the map, they don't exist.  Same for pylons and the like.  I understand some things are supposed to be hidden behind progression, but Dynmap not showing things you can see without any progression seems wrong.

I'd just like to be able to show off my world to people.

**Comment by Rubyheart** [comment]
https://www.reddit.com/r/feedthebeast/comments/1118p6t/comment/j8e5d4b/?utm_source=share&utm_medium=web2x&context=3

This link helped, but somethings aren't perfect.  Dye trees are fine, but rainbow trees just show as white, flower models are wonky, etc.

I'd still love to see native support.

**Comment by ReikaKalseki** [comment]
Without showing me how I am not even convinced it is possible, especially for things like blocks whose color/texture depends on code.

**Comment by Rubyheart** [comment]
It's beyond my understanding, otherwise I would have done it myself.

The reddit comment I linked to used Dynmap Scraper, who's documentation is [here](https://github.com/webbukkit/dynmap/wiki/Texture-Definition-Files).

Yet another computer died on me though, so I'm unable to actually play again for the time being.

**Comment by ReikaKalseki** [comment]
Do you know of any other mods with such integration?

**Comment by Rubyheart** [comment]
If I remember correctly, Dynmap really does not like things like pipes and wires, such as from AE2 or IC2, or some Botania flowers.  Most Rotarycraft machines are just solid blocks.

I think there's versions of the scraper that work for dynmap better, but only after 1.7.10.

The files in the comment change the render a bit, like the leaves are on the trees, just the wrong color.  I guess I'm not understanding your question enough to give you more useful information.

**Comment by ReikaKalseki** [comment]
As in other mods that inject this kind of data.

**Comment by Rubyheart** [comment]
I don't know of any other mods that dynamically provide textures to an online isometric map.  I don't think many mods to exist to provide maps in this way.

**Comment by ReikaKalseki** [comment]
No, I mean to dynmap.

**Comment by Rubyheart** [comment]
I don't know of any other mods that provide data to dynmap other than the dnymap scraper, which is not always required.
